src/ResourceManager/MachineLearningCompute/Commands.MachineLearningCompute/packages.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Management.MachineLearningCompute" version="0.1.0" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
@shutchings remove same packages outlined in csproj

Reply:
Done.